.github/workflows/tests.yaml

```diff
@@ -9,21 +9,16 @@ on:
       - '**'
 
 env:
-  BEE_VERSION: '1.4.1-238867f1-stateful'
-  BLOCKCHAIN_VERSION: '1.2.0-for-1.4.1-238867f1-stateful'
-  BEE_ENV_PREFIX: 'swarm-test'
-  BEE_IMAGE_PREFIX: 'docker.pkg.github.com/ethersphere/bee-factory'
-  COMMIT_VERSION_TAG: 'false'
-  WORKERS: 2
-  BEE_API_URL: 'http://localhost:1633'
+  BEE_API_URL: 'http://127.0.0.1:1633'
+  BEE_DEBUG_API_URL: 'http://127.0.0.1:1635'
 
 jobs:
   nodejs:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x, 17.x]
```

Comment:
What about adding Node 18?

Reply:
Even on bee-js we don't test against node v 18. I tried it but te orchestraction fails